### PR TITLE
getDeviceDatabase method added

### PR DESCRIPTION
**Pull request by aleks-pro**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -28,5 +29,6 @@ export default {
     generateThumbnail,
     isValidDeviceName,
     watchWindow,
-    bringWindowToFront
+    bringWindowToFront,
+    getDeviceDatabase
```
Comment (AndreyBelym) [inline src/index.js]: Maybe let's call it `getDevicesViewportsSet`?

\cc @VasilyStrelyaev

Comment (VasilyStrelyaev) [reply src/index.js]: maybe `getDevicesViewportData` ?

Comment (AndreyBelym) [reply src/index.js]: Sounds good

File: src/api/get-device-database.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { readSync } from 'read-file-relative';
+
+
+const DEVICE_DATABASE = JSON.parse(readSync('../../data/devices.json'));
+
+ /** Gets the name and the viewport size of all devices from [this table]{@link http://viewportsizes.com/}.
+ * @function
+ * @name getDeviceDatabase
+ * @returns {DeviceDatabase} JSON Object which contains devices name and viewport size.
```
Comment (VasilyStrelyaev) [inline src/api/get-device-database.js]:
```
 * @returns {DeviceDatabase} A JSON Object that contains device names and viewport sizes.
```

**Comment by aleks-pro** [comment]
FPR

File: src/api/get-devices-viewport-data.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { readSync } from 'read-file-relative';
+
+
+const VIEWPORT_DATA = JSON.parse(readSync('../../data/devices.json'));
+
+ /** Gets the name and the viewport size of all devices from [this table]{@link http://viewportsizes.com/}.
+ * @function
+ * @name getDevicesViewportData
+ * @returns {ViewportData} JSON Object which contains devices name and viewport size.
```
Comment (VasilyStrelyaev) [inline src/api/get-devices-viewport-data.js]: Please change the description according to https://github.com/DevExpress/testcafe-browser-tools/pull/147#discussion_r172150601

**Comment by aleks-pro** [comment]
FPR

**Pull request merged**